spec/jobs/rapid_ready_for_decision/form526_hypertension_job_spec.rb

```diff
@@ -0,0 +1,186 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sidekiq/testing'
+
+RSpec.describe RapidReadyForDecision::Form526HypertensionJob, type: :worker do
```

Comment:
This file is a copy of `DisabilityCompensationJob`'s rspec with updates only to the class name.
Running `diff spec/jobs/rapid_ready_for_decision/disability_compensation_job_spec.rb spec/jobs/rapid_ready_for_decision/form526_hypertension_job_spec.rb` will show only name changes.